lib/Tab.js

```diff
@@ -299,11 +299,12 @@ const styles = StyleSheet.create({
     flex: 1,
     alignItems: 'center',
     paddingTop: 8,
-    paddingBottom: 10,
+    paddingBottom: (Platform.OS !== 'web' ? 10 : 0),
```

Comment:
Web sets overflow hidden for all texts and there's not enough room to display it properly. This looks a bit odd but works